azerothcore/azerothcore-wotlk
# fix(Core/Spells): Gor'drek's Ointment

## Changes Proposed:

This PR proposes changes to:

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/5227
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/18475

## SOURCE:

The changes have been validated through:

https://www.youtube.com/watch?v=f6zqIJHKa48

## Tests Performed:

This PR has been:


## How to Test the Changes:



.q add 10488
.go c id 20748
Use the item, you get credit
50% chance to enter combat .gm on if you enter
aura should stay on the creature
.gm off, try to use item again on the creature
shouldn't allow you
Try using item on other npcs


## Known Issues and TODO List:


- [ ] https://github.com/azerothcore/azerothcore-wotlk/issues/19246


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
